bliss/train.py

```diff
@@ -44,10 +45,15 @@ def train(cfg: DictConfig):  # pylint: disable=too-many-branches
 
     # setup trainer
     logger = setup_logger(cfg, paths)
-    checkpoint_callback = setup_callbacks(cfg)
+    checkpoint_callback = setup_checkpoint_callback(cfg)
+    early_stopping = setup_early_stopping(cfg)
     profiler = setup_profiler(cfg)
 
-    callbacks = [] if checkpoint_callback is None else [checkpoint_callback]
+    callbacks = []
+    if checkpoint_callback:
+        callbacks.append(checkpoint_callback)
+    if early_stopping:
+        callbacks.append(early_stopping)
```

Comment:
Perhaps add a test for early stopping? Both to silence this warning, and also as good practice.

Reply:
Imo it would be tricky to actually test early stopping itself, since it would need to run for at least a couple of epochs and would cause tests to take a lot longer. Instead, I just toggled `enable_early_stopping` in `testing_config` so that the callback initialization gets hit in `test_train`.